clouddq/utils.py

```diff
@@ -101,6 +101,26 @@ def write_templated_file_to_path(path: Path, lookup_table: typing.Dict) -> None:
     path.write_text(get_template_file(lookup_table.get(path.name)))
 
 
+def set_dbt_intermediate_expiration_hours(
```

Comment:
I don't really like this method here. It feels very hacky to replace a string on a file format that was created from a class that contains the field we are trying to replace. Can we not find another way to override the expiration hours on the __init__ method of DbtRunner?

Reply:
We are writing the templated file that contains expiration hours. So I am updating the templated file content before writing to the target location. This processing is done inside **init** method of DbtRunner. We can chat more.